app/views.py

```diff
@@ -11,27 +11,56 @@
 from django.views.generic.edit import FormView, UpdateView
 
 from django.contrib.contenttypes.models import ContentType
```

Comment:
I know you didn't write this line, but you can merge all of these with the above imports (everything that *doesn't* start with `app.`), thanks.